homeassistant/components/persistent_notification/__init__.py

```diff
@@ -105,6 +108,10 @@ def async_create(
         UpdateType.ADDED,
         {notification_id: notifications[notification_id]},
     )
+    hass.bus.fire(
```

Comment:
This is using the sync api in async context

Reply:
Good call, fixed that.